PRODUCTIONISATION.md

```diff
@@ -126,13 +128,13 @@ To be applied if infrastructure has been orchestrated and provisioned specifical
 
 ### 10. Dependency upgrade alerts
 
-* 10.1 [ ] Configure a dependency upgrade notification service to submit pull requests to VCS service repository with alerts
+* [ ] 10.1 Configure a dependency upgrade notification service to submit pull requests to VCS service repository with alerts
 
 ## Platform specific
 
 * [Ruby][ruby_reference]
 
-## Recommended services
+## Suggested services
```

Comment:
Again, just curious why the need for this word change?

Reply:
We plan on adding more services and some might be better than others depending on the project.